### ISSUE TITLE
Can't register more than one class?

### ISSUE THREAD

**Issue by Des-Nerger** [thimenesup/GodotZigBindings#12]
Consider this (rudimentary) project: https://github.com/Des-Nerger/GodotZigBindings-demo-projects/tree/cc02d07/dodge_the_creeps. It has a player scene with a class defined in a Zig GDExtension; all work fine. However, if I copy-paste this same class, giving it a different name:
```sh
$ sed 's/Player/ExactSamePlayer/' player.zig >exact_same_player.zig
```
And then try to register it along with the old one:
```diff
--- a/dodge_the_creeps/lib.zig
+++ b/dodge_the_creeps/lib.zig
@@ -11,4 +11,5 @@ fn gdextension_initialize(init_level: gi.GDExtensionInitializationLevel) callcon
         inline for (.{
+            @import("exact_same_player.zig").ExactSamePlayer,
             @import("player.zig").Player,
         }) |user_class|
             ClassDB.registerClass(user_class, false, false); // _, is_virtual, is_abstract
```
, then the old one just stops working: the game doesn't react to keyboard input anymore, etc.
Do you have an idea why this is happening?

**Comment by Des-Nerger** [comment]
Hm, interesting. Looks like the order of the imports is significant. If I import `Player` first, and only after that do `ExactSamePlayer`, then the things don't break out. :thinking:

**Comment by thimenesup** [comment]
Fixed by 4819e3d.
